src/components/IstioWizards/IstioWizard.tsx

```diff
@@ -234,18 +233,8 @@ class IstioWizard extends React.Component<Props, State> {
 
   onCreate = () => {
     const [dr, vr] = this.createIstioTraffic();
-    const createDR = API.createIstioConfigDetail(
-      authentication(),
-      this.props.namespace,
-      'destinationrules',
-      JSON.stringify(dr)
-    );
-    const createVS = API.createIstioConfigDetail(
-      authentication(),
-      this.props.namespace,
-      'virtualservices',
-      JSON.stringify(vr)
-    );
+    const createDR = API.createIstioConfigDetail(this.props.namespace, 'destinationrules', JSON.stringify(dr));
```

Comment:
I guess team Jupiter should be aware of the API changes.
CC @lucasponce @aljesusg @xeviknal @jotak

Reply:
It doesn't change the logic.
I assume that the authentication() is refactored.
@hhovsepy perhaps you could run some regression test on this PR ?
Thanks a lot.